demos/object_detection_demo_ssd_async/CMakeLists.txt

```diff
@@ -2,10 +2,14 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+FILE(GLOB SRC_FILES ./*.cpp ../common/processing/*.cpp)
+FILE(GLOB H_FILES ./*.h ../common/processing/*.h)
```

Comment:
Please don't do that. If the files are in `common`, they should belong to the `common` target (or, alternatively, to a separate library).

Reply:
preprocessing is converted to separate library, similar to monitors

Reply:
id that model_den folder now? What means that name? An it is not added to CMakeLists, so sample now can't be built

Reply:
changed back to processing